From a654b9adc50f00c70f118cc8c1c87abca4b987e9 Mon Sep 17 00:00:00 2001
From: Adri <git@verbto.be>
Date: Mon, 26 Oct 2026 18:17:46 +0100
Subject: [PATCH 2/2] a22x only: drop the vendor.camera.aux.packagelist
 property rule, the stock Samsung vendor defines it with another type and init
 cannot build property_info

Change-Id: I478bda0fd5625fb910e8f08a115ed475ccc6d3b1
---
 common/private/property_contexts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/common/private/property_contexts b/common/private/property_contexts
index 9bdd72f..52787d2 100644
--- a/common/private/property_contexts
+++ b/common/private/property_contexts
@@ -4,7 +4,6 @@ ro.odm.cast.ssid_suffix u:object_r:odm_cast_prop:s0
 
 # Aux camera
 vendor.camera.aux.packageexcludelist   u:object_r:vendor_persist_camera_prop:s0
-vendor.camera.aux.packagelist          u:object_r:vendor_persist_camera_prop:s0
 
 # Bluetooth
 bluetooth.hci.disabled_commands        u:object_r:bluetooth_config_prop:s0
-- 
2.47.3

